include/logger_lib.hpp

```diff
@@ -0,0 +1,8 @@
+#ifndef LOGGER_LIB_HPP
+#define LOGGER_LIB_HPP
+
+namespace logger
+{
+};
```

Comment:
This file must be in folder include/Logger 

So after installation it will not mix with the headers N and path resolution will also not affected for internal includes..